package.json

```diff
@@ -184,3 +184,4 @@
     }
   }
 }
+
```

Comment:
Please ... I don't want to mess with linters or whatever, so please just leave this untouched. There has to be one empty line at the end. There is no need to change on this eof for this PR.

Reply:
It looks like the file in master doesnt have the empty line at the end